### PR TITLE
qa-jcoin Homepage and explorer UI final content changes

### PR DESCRIPTION
**Pull request by mbkranz**
Link to Jira ticket if there is one:

### Environments
QA

#### Homepage changes
- Added more specific JCOIN title/description
- add total enrolled bar chart (using the new enrollment node)

#### Explorer
- OEPS tab: remove (will make available under Datasets tab
- Files tab: change name to "datasets" 
- Removed unused features to reduce complexity/confusion of UI : notebook browser and private hub documentation 
    (putting private hub docs in public docs and notebook browsers --but kept resource and set to private for reference -- not used for this research community)

Additionla features will be added in portal (e.g., study and publication viewer) and potentially minor style changes (ie HEAL colors. But not for 1-2 months

### CODE REVIEW

File: qa-jcoin.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -82,25 +87,14 @@
...
         },
-        {
-          "icon": "dictionary",
-          "link": "https://qa-jcoin.planx-pla.net/dashboard/Secure/index.html",
-          "color": "#a2a2a2",
-          "name": "HUB Documentation"
-        },
-        {
-          "name": "Notebook Browser",
-          "link": "/resource-browser",
```
Comment (paulineribeyre) [inline qa-jcoin.planx-pla.net/portal/gitops.json:101]: removing the link to the MOUD notebook?

Comment (mbkranz) [reply qa-jcoin.planx-pla.net/portal/gitops.json:101]: @paulineribeyre  Does this sound ok?

Comment (paulineribeyre) [reply qa-jcoin.planx-pla.net/portal/gitops.json:101]: ok

Comment (paulineribeyre) [reply qa-jcoin.planx-pla.net/portal/gitops.json:101]: @mbkranz should we hold off merging until Phil has a look?

**Review by paulineribeyre — APPROVED** [review]

**Comment by mbkranz** [comment]
@paulineribeyre  thanks Pauline, ready for a merge!

**Pull request merged**